test/unit/config_tests.rb

```diff
@@ -0,0 +1,259 @@
+require 'assert'
+require 'dk-dumpdb/config'
+
+require 'dk-dumpdb/db'
+# require 'test/support/test_scripts'
+
+class Dk::Dumpdb::Config
+
+  class UnitTests < Assert::Context
+    desc "Dk::Dumpdb::Config"
+    setup do
+      @script = FakeScript.new
+      @config_class = Dk::Dumpdb::Config
+    end
+
+  end
+
+  class InitTests < UnitTests
+    desc "when init"
+    setup do
+      now = Time.now
```

Comment:
@kellyredding - Really minor, you might use `Factory.time` if the time value doesn't really matter because it should choose more random values from the past and future.